src/applications/appeals/995/tests/components/EvidencePrivateRecordsRequest.unit.spec.jsx

```diff
@@ -65,6 +65,21 @@ describe('<EvidencePrivateRecordsRequest>', () => {
     expect(goSpy.called).to.be.false;
   });
 
+  it('should submit page', () => {
+    const goSpy = sinon.spy();
+    const data = { [EVIDENCE_PRIVATE]: true };
+    const { container } = render(
+      <div>
+        <EvidencePrivateRecordsRequest data={data} goForward={goSpy} />
```

Comment:
Is pressing the next button the action that checks the page is "submitted"?

Reply:
Yes